algo/src/test/kotlin/com/hartwig/actin/algo/evaluation/molecular/ProteinExpressionByIHCFunctionsTest.kt

```diff
@@ -5,23 +5,42 @@ import com.hartwig.actin.algo.evaluation.molecular.MolecularTestFactory.priorIHC
 import com.hartwig.actin.algo.evaluation.util.ValueComparison
 import com.hartwig.actin.datamodel.PatientRecord
 import com.hartwig.actin.datamodel.algo.EvaluationResult
+import com.hartwig.actin.datamodel.molecular.MolecularHistory
+import com.hartwig.actin.datamodel.molecular.TestMolecularFactory
+import org.assertj.core.api.Assertions.assertThat
 import org.junit.Test
 
 private const val IHC = "IHC"
 private const val PROTEIN = "PD-L1"
+private const val GENE = "PD-L1"
```

Comment:
I would use `CD274` here since that is the gene that produced PD-L1 protein. It doesn't actually matter for the test but it does look weird in my brain to not see `CD274` is you already use a real protein as well! 

EDIT: I see that all other tests use fake proteins and genes, maybe switch to fake in this test as well if possible?